content/src/roadmaps/license-tasks/apply-for-food-truck-license.md

```diff
@@ -11,13 +11,12 @@ webflowType: business-license
 webflowId: 64120ab148da55fdb9c3b1c4
 localLevelTask: Local Department of Health
 licenseCertificationClassification: LICENSE
+summaryDescriptionMd: "Food trucks need to be cleaned and sanitized according to the guidelines from your local Department of Health. The regulation and licensing of food trucks vary by location.
+You'll need to check license requirements with the county and/or local Department of Health in every county where you plan to do business."
 ---
-Food trucks need to be cleaned and sanitized according to the guidelines from your local Department of Health. The regulation and licensing of food trucks vary by location. 
 
-You'll need to check license requirements with the county and/or local Department of Health in every county where you plan to do business.
-
-- - -
+---
```

Comment:
I can't tell if this is the diff playing tricks on my eyes or not. We only need one `---` right? Is this 2 in a row?